WordPress/src/main/java/org/wordpress/android/ui/media/WordPressMediaUtils.java

```diff
@@ -280,6 +280,22 @@ public static Cursor getWordPressMediaVideos(String blogId) {
     }
 
     /**
+     * Given a VideoPress id, returns the corresponding remote video URL stored in the DB
+     * @param blogId the current blog id
+     * @param videoPressId the VideoPress id (used in VP shortcodes: [wpvideo the-VideoPress-id])
+     */
+    public static String getNetworkVideoUrlFromVideoPressId(String blogId, String videoPressId) {
+        Cursor cursor = WordPress.wpDB.getMediaFileByVideoPressId(blogId, videoPressId);
+
+        String path = "";
+        if (cursor != null && cursor.moveToFirst()) {
+            path = cursor.getString(cursor.getColumnIndex("fileURL"));
```

Comment:
Rather than hard-code `fileURL` here, it should use `WordPressDB.COLUMN_NAME_FILE_URL`.